Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/volunteer/VolunteerAdapterItem.kt

```diff
@@ -35,12 +36,8 @@ class VolunteerAdapterItem internal constructor(val itemData: VolunteerEvent) :
 
         val context = holder.name.context
 
-        Glide.with(context)
-                .load(itemData.pictureUrl)
-                .transform(CircleTransform(context))
-                .placeholder(R.drawable.emo_im_cool)
-                .crossFade()
-                .into(holder.avatar)
+        Glide.with(context).load(itemData.pictureUrl).transform(CircleTransform(context))
+                .placeholder(R.drawable.emo_im_cool).crossFade().into(holder.avatar)
```

Comment:
Like above, chain can stay line-by-line but be lined up with `.with()`.

Reply:
@redbassett Do you know the exact rule to set that in the code style?

Reply:
I don't, but I've never had the auto formatter fix it for me? The formatter can't really tell what is a chain and what is just methods called on results. I'm fine breaking with formatter suggestion for this.

Reply:
i'd like to use the default code formatter for kotlin if possible.

Reply:
I prefer code formatter because it keeps everyone on the same page. We agree to have one convention by hand and then someone else breaks it because it is not enforced. 

@kimj that is the one enforced here.

Reply:
Sticking to official kotlin style guide.